packages/tests/e2e/robot/resources/keywords.robot

```diff
@@ -138,6 +138,19 @@ Cleanup resource
     [Arguments]  ${entity type}  ${entity id}
     StrictShell  odahuflowctl --verbose ${entity type} delete --id ${entity id} --ignore-not-found
 
+E2E test setup
+    [Arguments]  ${example_id}
+    Set Environment Variable  ODAHUFLOW_CONFIG  ${LOCAL_CONFIG}
+    Login to the api and edge
+    Cleanup example resources  ${example_id}
+
+E2E test teardown
+    [Arguments]  ${example_id}
+    ${NO_CLEAN_UP_lower}  Evaluate  '${NO_CLEAN_UP}'.lower()
+    Pass Execution If  '${NO_CLEAN_UP_lower}' != 'false'  Suite Teardown is not executed
```

Comment:
Is there a way to operate with `NO_CLEAN_UP` parameter as `bool`, not `string`? If yes, please implement it that way.

Reply:
OK, I'll try. If it's not possible I will add the comment in `keywords.robot` file.